.circleci/config.yml

```diff
@@ -39,10 +46,10 @@ workflows:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
       - build-image:
+          context: falco
           requires:
             - test
             - lint
           filters:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
```suggestion
            branches:
              only: "master"
```
If we want to push the `lastest` tag only when the master has been updated, I think we have to filter just for that

Reply:
we also want to just to build in PRs to make sure the image can be built

Reply:
So we want to build on every branch, but push the image for the master only?

Reply:
we go with two jobs as discussed in slack